php/EE/Migration/Containers.php

```diff
@@ -0,0 +1,150 @@
+<?php
+
+namespace EE\Migration;
+use EE\RevertableStepProcessor;
+use function EE\Utils\default_launch;
+use EE;
+
+/**
+ * Migrates existing containers to new image
+ */
+class Containers {
+
+	/** @var RevertableStepProcessor */
+	private static $rsp;
```

Comment:
Add a proper comment like you did other places: https://github.com/EasyEngine/easyengine/pull/1134/files#diff-30cb40e291ec951baeed05a3d98ffd6cR14

Try to maintain code consistency.